services/fileServices/MdPageServices/CollectionPageService.js

```diff
@@ -3,100 +3,114 @@ const {
   convertDataToMarkdown,
 } = require("@utils/markdown-utils")
 
-const GitHubService = require("@services/db/GitHubService")
-const CollectionYmlService = require("@services/fileServices/YmlFileServices/CollectionYmlService")
+class CollectionPageService {
+  constructor({ gitHubService, collectionYmlService }) {
+    this.GitHubService = gitHubService
+    this.CollectionYmlService = collectionYmlService
+  }
 
-const Create = async (
-  reqDetails,
-  { fileName, collectionName, content, frontMatter }
-) => {
-  const parsedCollectionName = `_${collectionName}`
+  async Create(reqDetails, { fileName, collectionName, content, frontMatter }) {
+    const parsedCollectionName = `_${collectionName}`
 
-  await CollectionYmlService.AddItemToOrder(reqDetails, {
-    collectionName,
-    item: fileName,
-  })
+    await this.CollectionYmlService.AddItemToOrder(reqDetails, {
```

Comment:
Could we use `camelCase` for both the **instance** as well as the **method** of a class?
i.e. `this.collectionYmlService.addItemToOrder`

The use of `PascalCase` suggests (at least to me) that it is a class name or constructor. Thoughts?

Reply:
Addressed in https://github.com/isomerpages/isomercms-backend/pull/253